codingtest_ajw/4일차/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,18 @@
+function solution(numbers) {
+    let answer = [];
+   
+    for(let i = 0; i < numbers.length; i++){
+        for(let j = i + 1; j < numbers.length; j++){
+          
+             let inumindex = numbers[i];
+             let jnumindex = numbers[j];
+             answer.push(inumindex + jnumindex);
```

Comment:
```suggestion
             answer.push(numbers[i] + numbers[j]);
```
불필요한 변수 선언은 하지 않는게 좋습니다.